evalml/tests/component_tests/test_prophet_regressor.py

```diff
@@ -0,0 +1,154 @@
+import numpy as np
```

Comment:
There are multiple tests taking about ~15 seconds. Can we tweak the prophet parameters to make them go faster. The data is already small enough lol 😨 

```
=========================================== slowest durations ===========================================
18.53s call     evalml/tests/component_tests/test_prophet_regressor.py::test_fit_predict_ts_with_X_index
17.10s call     evalml/tests/component_tests/test_prophet_regressor.py::test_fit_predict_date_col
16.45s call     evalml/tests/component_tests/test_prophet_regressor.py::test_fit_predict_ts_with_y_index
15.55s call     evalml/tests/component_tests/test_prophet_regressor.py::test_fit_predict_ts_no_X
7.60s call     evalml/tests/component_tests/test_prophet_regressor.py::test_feature_importance
0.01s call     evalml/tests/component_tests/test_prophet_regressor.py::test_cmdstanpy_backend
```

Reply:
@freddyaboulton @chukarsten By changing the default parameters for `Prophet` to
`prophet.Prophet(uncertainty_samples=False, changepoint_prior_scale=2.0)`, 
the new times are
```
=========================================== slowest durations ===========================================
7.72s call     evalml/tests/component_tests/test_prophet_regressor.py::test_fit_predict_ts_with_y_index
6.34s call     evalml/tests/component_tests/test_prophet_regressor.py::test_fit_predict_ts_no_X
4.25s call     evalml/tests/component_tests/test_prophet_regressor.py::test_fit_predict_date_col
3.85s call     evalml/tests/component_tests/test_prophet_regressor.py::test_feature_importance
3.26s call     evalml/tests/component_tests/test_prophet_regressor.py::test_fit_predict_ts_with_X_index
```